src/common/config.js

```diff
@@ -63,7 +63,7 @@ const Config = {
 
   INTERVALS: {
     TELEMETRY_FLOW_DURATION: 1500,
-    TELEMETRY_UPDATE_INTERVAL: 2500,
+    TELEMETRY_UPDATE_INTERVAL: 15000,
```

Comment:
This block already has INTERVALS at the top level, so may be just TELEMETRY_UPDATE to avoid redundancy